hax/hax/handler.py

```diff
@@ -4,7 +4,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#    http://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/lic://github.com/Seagate/cortx-motr/blob/b22c3dd234b7885b6528e8870192f00816d75048/pool/pool.c#L659enses/LICENSE-2.0
```

Comment:
Is this hyperlink correct?

Reply:
Seems like motr's pool.c link mistakenly got copied here

Reply:
Damn, must have got pasted mistakenly from the cache, restored.